charts/chart_deps/fluent/fluentbit/fluentbit-standalone/templates/secret.yaml

```diff
@@ -1,5 +1,11 @@
-{{- range $name, $secret := .Values.secrets }}
+{{- if .Values.enabled }}
```

Comment:
i think we need to remove this because we cant use standalone chart without creating a fluentbit secret